reach

```diff
@@ -115,10 +115,143 @@ run_s () {
   sh -ac "$TMP/out.sh" "$0"
 }
 
+gen_reachc_id () {
+  REACH_ENV_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/reach/"
+  REACH_ENV_FILE="${REACH_ENV_DIR}/env"
+  MD5_CMD=md5sum||md5
+  PRE_REACHC_ID=$(head -n1 /dev/urandom | $MD5_CMD | head -c 30 | tr [a-z] [A-Z])
+
+  mkdir -p $REACH_ENV_DIR
+
+  if [ ! -e $REACH_ENV_FILE ]; then
+    echo "export REACHC_ID=${PRE_REACHC_ID}" > $REACH_ENV_FILE
+  elif ( ! grep REACHC_ID $REACH_ENV_FILE > /dev/null 2>&1 ); then
+    echo "export REACHC_ID=${PRE_REACHC_ID}" >> $REACH_ENV_FILE
+  fi
+  . $REACH_ENV_FILE
+}
+
+call_home () {
+  START_TIME=$1
+  RESULT=$2
+  REACH_VERSION=$3
+  CONNECTOR_MODE=$4
+  USING_VSSTUDIO_EXTENTION=$5
+  INITIATOR=$6
+
+  CALL_HOME_ENCODED="eyAidXNlcklkIjogInt7UkVBQ0hDX0lEfX0iLCAic3RhcnRUaW1lIjogInt7U1RBUlRfVElNRX19\
+IiwgInJlc3VsdCI6ICJ7e1JFU1VMVH19IiwgInZlcnNpb24iOiAie3tSRUFDSF9WRVJTSU9OfX0i\
+LCAiY29ubmVjdG9yTW9kZSI6ICJ7e0NPTk5FQ1RPUl9NT0RFfX0iLCAidXNpbmdWaXN1YWxTdHVk\
+aW9FeHRlbnNpb24iOiAie3tVU0lOR19WU1NUVURJT19FWFRFTlRJT059fSIsICJpbml0aWF0b3Ii\
+OiAie3tJTklUSUFUT1J9fSIgfQ==" ###Change the template to not track elapsed time
+  CALL_HOME_TEMPLATE_DECODED=$(echo -n $CALL_HOME_ENCODED | base64 -d)
+  START_TIME=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+  TRANSLATED_TEMPLATE=$(echo $CALL_HOME_TEMPLATE_DECODED | sed -e "s/{{REACHC_ID}}/$REACHC_ID/g" \
+  -e "s/{{START_TIME}}/${START_TIME}/g" \
+  -e "s/{{RESULT}}/${RESULT}/g" \
+  -e "s/{{REACH_VERSION}}/${REACH_VERSION}/g" \
+  -e "s/{{CONNECTOR_MODE}}/${CONNECTOR_MODE}/g" \
+  -e "s/{{USING_VSSTUDIO_EXTENTION}}/${USING_VSSTUDIO_EXTENTION}/g" \
+  -e "s/{{INITIATOR}}/${INITIATOR}/g")
+  CALL_HOME_ENDPOINT="https://log.reach.sh/submit"
```

Comment:
```suggestion
```

Reply:
I have removed this variable. This was only been used on curl.